bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -37,14 +41,69 @@ def __init__(self, bot: Bot) -> None:
+        self.bot.loop.create_task(self.schedule_autoreviews())
+
+    async def schedule_autoreviews(self) -> None:
+        """Reschedule reviews for active nominations if autoreview is enabled."""
+        if await self.autoreview_enabled():
+            await self.reviewer.reschedule_reviews()
+        else:
+            self.log.trace("Not scheduling reviews as autoreview is disabled.")
+
+    async def autoreview_enabled(self) -> bool:
+        """Return whether automatic posting of nomination reviews is enabled."""
+        return await self.talentpool_settings.get(AUTOREVIEW_ENABLED_KEY, True)
 
     @group(name='talentpool', aliases=('tp', 'talent', 'nomination', 'n'), invoke_without_command=True)
     @has_any_role(*MODERATION_ROLES)
     async def nomination_group(self, ctx: Context) -> None:
         """Highlights the activity of helper nominees by relaying their messages to the talent pool channel."""
         await ctx.send_help(ctx.command)
 
+    @nomination_group.group(name="autoreview", aliases=("ar",), invoke_without_command=True)
+    @has_any_role(*MODERATION_ROLES)
+    async def nomination_autoreview_group(self, ctx: Context) -> None:
+        """Commands for enabling or disabling autoreview."""
+        await ctx.send_help(ctx.command)
+
+    @nomination_autoreview_group.command(name="enable", aliases=("on",))
+    @has_any_role(Roles.admins)
+    async def autoreview_enable(self, ctx: Context) -> None:
+        """
+        Enable automatic posting of reviews.
+
+        This will post reviews up to one day overdue. Older nominations can be
+        manually reviewed with the `tp post_review <user_id>` command.
+        """
+        if await self.autoreview_enabled():
+            await ctx.send(":x: Autoreview is already enabled")
+            return
+
+        await self.talentpool_settings.set(AUTOREVIEW_ENABLED_KEY, True)
+        await self.reviewer.reschedule_reviews()
+        await ctx.send(":white_check_mark: Autoreview enabled")
+
+    @nomination_autoreview_group.command(name="disable", aliases=("off",))
+    @has_any_role(Roles.admins)
+    async def autoreview_disable(self, ctx: Context) -> None:
+        """Disable automatic posting of reviews."""
+        if not await self.autoreview_enabled():
+            await ctx.send(":x: Autoreview is already disabled")
+            return
+
+        await self.talentpool_settings.set(AUTOREVIEW_ENABLED_KEY, False)
+        self.reviewer.cancel_all()
+        await ctx.send(":white_check_mark: Autoreview disabled")
+
+    @nomination_autoreview_group.command(name="status")
+    @has_any_role(*MODERATION_ROLES)
+    async def autoreview_status(self, ctx: Context) -> None:
+        """Show whether automatic posting of reviews is enabled or disabled."""
+        if await self.autoreview_enabled():
+            await ctx.send("Autoreview is currently enabled")
+        else:
+            await ctx.send("Autoreview is currently disabled")
```

Comment:
```suggestion
        """Show whether automatic posting of reviews is enabled or disabled."""
        status = "enabled" if await self.autoreview_enabled() else "disabled"
        await ctx.send(f"Autoreview is currently {status}")
```

Reply:
Neither seems preferable over the other to me. Your suggestion looks neater but the one liner is maybe slightly less readable.